### DISCUSSION TITLE
New open! Academic conference 2022

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#79] [General]
작년에 이어 올해도 academic conference(한글명: 학술회의) 2022을 개최합니다.

작년 참여 멤버들의 의견을 적극 반영하여 진행할 예정이고
작년 참여 멤버 외에 새롭게 들어온 멤버들이 있습니다.

@yoonucho (2021 member)
@hi-hyein (2021 member)
@Joseph-Cha (2021 member)
@wisdom08 
@evelon 
@momoci99 
@TaeHyoungKwon 
@wooyaggo86 

아래 내용들은 default case 입니다.
추가 의견에 대해서는 댓글로 남겨주세요.

결정이 진행되는 방식은 `허락 보다 용서가 쉽다` 입니다.
의견을 제시하는 분에 맞춰서 계획은 세워지되,
안되거나 다른 의견이 있는 분이 대안을 제시하고 그 의견이 반영되는 방식입니다.

---

### Date & Time

- 2주 간격 금요일 오후 11시이고 예정된 일자는 아래와 같습니다.
- 1월 1일, 책 읽기 시작!
- 1월 14일, 함께 자라기 1 자라기
- 1월 28일, 함께 자라기 2 함께
- 2월 11일, 함께 자라기 3 애자일
- 2월 25일, 프로그래머의 길 멘토에게 묻다 1장, 2장
- 3월 11일, 프로그래머의 길 멘토에게 묻다 3장, 4장
- 3월 25일, 프로그래머의 길 멘토에게 묻다 5장, 6장
- 4월 8일, 프로그래머의 길 멘토에게 묻다 7장 + 실용주의 프로그래머 1장
- 4월 22일, 실용주의 프로그래머 2장 (66p)
- 5월 6일, 실용주의 프로그래머 3장, 4장 (78p)
- 5월 20일, 실용주의 프로그래머 5장, 6장 (92p)
- 6월 3일, 실용주의 프로그래머 7장 (76p)
- 6월 17일, 실용주의 프로그래머 8장, 9장, 맺는말 (62p)
- 6월 24일(금요일) ~ 25일(토요일), 워크샵 예정

책 읽는 순서를 바꾸고 싶은 분은 의견 주세요.

### Adjustment

- 위에 적힌 날짜가 미팅 일정입니다. 이 날 개인 사정이 있으신 분은 얘기하시고 대안 날짜와 시간을 얘기해 주세요.
- 아래 예시가 `허락 보다 용서가 쉽다`의 예시입니다.
- 예) 1월 28일은 설날 연휴이므로 1월 27일에 했으면 좋겠습니다 => 1월 27일이 안되는 분은 대안 날짜를 얘기해 주셔야 하며, 아무 얘기 없으면 그대로 1월 27일로 갑니다.
- 예2) 3월 11일이 아내 생일이어서 3월 12일 토요일 오후 9시로 했으면 좋겠습니다. => 3월 12일이 안되는 분은 대안 날짜 및 시간을 얘기해 주셔야 하며, 역시 아무 얘기 없으면 3월 12일 토요일 오후 9시로 진행합니다.

### Detail rules

- 약 2주간 정해진 분량의 책을 읽고, 리뷰 D-3일 전까지 논의할 내용 pull request로 등록
  - 미팅 이틀 전인 수요일 23:59까지 자유롭게 공개, 되도록 수요일 오후 시간 이후에 pull reqeust를 주세요
- 리뷰 모임 직전 까지 각 멤버별로 등록한 pull request에 코멘트를 달면서 논의 진행 완료
  - 코멘트를 적어도 하나 이상 달기를 권장합니다.
  - 최소한 approve를 해 줘야 리뷰를 했다는 인증이 됩니다.
- 미팅 날 온라인으로 실제 얘기하고 싶었던 부분에 대해 돌아가면서 짧게 얘기하며 진행
  - 산술적으로 한 멤버당 발언시간이 10분 남짓이 될 거 같습니다. 최소한 한 분 이상 발언할 시간 및 기회를 드리려고 합니다.
- 총 12회 반복해서 책 3권 완료, 마지막 13회는 워크샵 진행

**Comment by jongfeel** [comment]
책 읽기 시작하는 날은 새해 첫날 2022년 1월 1일 부터 입니다.

**Comment by jongfeel** [comment]
추가로 1월 7일 금요일 오후 11시 자기소개 미팅을 짧게 가지는 것도 좋을 것 같습니다.

**Comment by Joseph-Cha** [comment]
후.. 또 시작이 되는군요 ㅋㅋㅋㅋㅋ
일년의 절반을 써야해서 매우 부담이 크지만 그만큼 더 많이 남는 시간이 되었으면 좋겠네요!

**Reply by jongfeel** [reply]
꼭 그렇게 될 겁니다. 저번 처럼 빡세게(?) 할 거 같지는 않고요.
코멘트는 필요한 만큼 적당하게 다는 걸로 바꾸고, 리뷰 시간도 이틀로 줄였기 때문에 더 여유 있을 거에요.
